src/lib.rs

```diff
@@ -170,6 +172,23 @@ pub(crate) fn for_pairs(a: &Vec<IpAddr>, b: &Vec<IpAddr>, mut f: impl FnMut(IpAd
     }
 }
 
+/// Returns how long the currently executing host has been executing for in
+/// virtual time.
+///
+/// Must be called from within a Turmoil simulation.
+pub fn elapsed() -> Duration {
```

Comment:
While the docs are the same, this is definitely more readable in the source code.